### PR TITLE
Cf update morphic chapter

### PR DESCRIPTION
**Pull request by jecisc**
I updated the scripts/figures and fixed some things on the Morphic chapter.

### CODE REVIEW

File: Morphic/Morphic.pier [hunk 1]
```diff
@@ -323,21 +330,21 @@ all crossMorphs answer ==true== to the ==handlesMouseDown:== message.
 
 @@todo Add this method to ==CrossMorph==:
 
-[[[caption=Declaring that ==CrossMorph== will react to mouse clicks.
-CrossMorph>>>handlesMouseDown: anEvent
-	^true
+[[[language=smalltalk|caption=Declaring that ==CrossMorph== will react to mouse clicks|label=scr:mouseEvent
+CrossMorph>>#handlesMouseDown: anEvent
```
Comment (dmitrizagidulin) [inline Morphic/Morphic.pier]: I'm not sure we should switch our script convention here. (Or, if this is the right notation, we should go back and fix all other method definitions in the scripts).
I personally don't like the `>>>` notation that the book uses, but I understand it compiles into the special >> character.
So, I'm not necessarily arguing against `>>#`, but just pointing out we should be consistent.

**Comment by dmitrizagidulin** [comment]
Aside from a minor question about `>>#`, looks ready to merge. (+1) from me.

File: Morphic/Morphic.pier (hunk 1 above)
Comment (jecisc) [inline Morphic/Morphic.pier]: I think I saw that notation. But I never saw `>>>`. Two yes but not three. I'm not sure about the `#`, if we consider the name of the method as a symbol or not. I'll ask to Stephane what to use.

Comment (dmitrizagidulin) [inline Morphic/Morphic.pier]: Yeah, much of the book uses `>>>` in the method definition scripts. Pillar compiles it to a non-ascii character »
In any case, I'm merging the changes, we can sort out which notation we prefer, separately.

Comment (jecisc) [inline Morphic/Morphic.pier]: We can close issue https://github.com/SquareBracketAssociates/UpdatedPharoByExample/issues/85 with this PR.

**Pull request merged**